ports/filters.py

```diff
@@ -12,6 +12,14 @@ class Meta:
         fields = ['builder_name__name']
 
 
+class BuildHistoryFilterForPort(django_filters.FilterSet):
```

Comment:
> I'd need a little help in understanding this. How is this different from the BuildHistoryFilter just above?

`BuildHistoryFilterForPort` is definitely redundant. Thank you

> Also, I do not see the need to do a `contains` query in cases where the set of possible values for a field is fixed, e.g., status, builders in this case. An `exact` query should be performed instead.

We are also running a filter for "All Failed Builds" which includes all failed builds regardless of `failed install-port` or `failed install-dependencies`. So I just pass 'failed' and contains helps with that.

Reply:
While we do have an index on `status`, I am still unsure if the database will using it for a `contains` query and perform a full/partial table scan over all the builds while it will definitely be using the index when it's an `exact` query.

Irrespective of the above, matching fields like statuses should be exact anyway in my opinion. In this case, an appropriate value to be searched in the database could be tied to the "All Failed Builds". I need to think a little more on how we can implement it. What do you suggest, @mojca?

Reply:
Carried to https://github.com/macports/macports-webapp/issues/23